udf_utils.go

```diff
@@ -24,12 +28,28 @@ func (v pinnedValue[T]) unpin() {
 	v.pinner.Unpin()
 }
 
+func getPinnedValueValue[T any](handle unsafe.Pointer) T {
```

Comment:
Maybe `getPinned` is enough, and we do not have to decide between `ValueValue` and `Value`. 😄